iosApp/Survey/Sources/Presentation/Views/FadePaginationView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  FadePaginationView.swift
+//  Survey
+//
+//  Created by Bliss on 28/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct FadePaginationView<T>: View {
+
+    @Binding private var currentPage: Int
+    @State private var currentVisible: Double = 1.0
+    @State private var nextPage = 0
+    private var turnLength: Double = 160.0
+    private var turnSpeed: Double = 700.0
+    @GestureVelocity private var velocity: CGVector
```

Comment:
Can we have some in between line breaks for these variables for clean separation per type?